### DISCUSSION TITLE
Show spaces instead of underscores in markdown links

### DISCUSSION THREAD

**Discussion by aLily11** [vuepress/core#1000] [Q&A] [answered]
when I used [name] (link) , the difference in vuepress and Typora occured.
In Typora ,[title] (aaa_bbb) is  possible, and the " _ " will be excaped to space, the result is "aaa bbb"("aaa%20bbb" in url, %20 means space), it is the syntax of markdown
But in vuepress, it is still "aaa_bbb" in url and the " _ " isn't excaped, which makes difference with the url in sidebar(I modified markdown.slugify so it become original)
I want to the url is 'aaa bbb'  not 'aaa_bbb' , how could i do?

**Comment by Mister-Hope** [comment, marked as answer]
What I think is that typora is strange. It's not behaving the standard markdown convertion rule.

**Reply by aLily11** [reply]
fine. I'll use ' %20 ' instead of ' _ ' . thanks for answering my question.
